.editorconfig

```diff
@@ -0,0 +1,7 @@
+root = true
+
+# Used to enforce Sass styles in CI
+[*.scss]
```

Comment:
We can also add `quote_type` to choose are preference for string literal quote marks.

`quote_type = double`